barretenberg/exports.json

```diff
@@ -775,6 +815,46 @@
     ],
     "isAsync": false
   },
+  {
```

Comment:
This was missing @feltroidprime but I am not sure where it is used it at all

Reply:
This is a byproduct of running binding generation. did you?

Reply:
this is a compiler output that was committed when the binding code was made, it's not used, but shows that people ran binding gen

Reply:
No, I did not. I updated it manually. I will get to it then.

Reply:
@ludamad I ran bindgen and exposts.json did not change (the manual changes matched the expected output), however, the suffix ZKHonk for the API mehods were automatically replaced by ZkHonk, as you can see on this commit:

https://github.com/AztecProtocol/aztec-packages/pull/14372/commits/44f40ad0e98ffe09bb1012a20c9109464dec1456

Then I updated all references to also use ZkHonk:

https://github.com/AztecProtocol/aztec-packages/pull/14372/commits/eafe13b6446d2af8a569f2d43a9266c00c688693

Given that, I believe the PR is good to go. Let me know otherwise. Thanks!